src/features/DocumentFormatter.ts

```diff
@@ -384,6 +467,11 @@ export class DocumentFormatterFeature implements IFeature {
         this.rangeFormattingEditProvider = vscode.languages.registerDocumentRangeFormattingEditProvider(
             "powershell",
             this.documentFormattingEditProvider);
+        this.onTypeFormattingEditProvider = vscode.languages.registerOnTypeFormattingEditProvider(
+            "powershell",
+            this.documentFormattingEditProvider,
+            this.firstTriggerCharacter,
+            ...this.moreTriggerCharacters);
```

Comment:
This is such a bizarre way for them to define this API...  If it's a new API why have a "first" param and then a "more" param?  Weird.